resources/assets/lib/models/chat/message.ts

```diff
@@ -38,6 +38,11 @@ export default class Message {
     this.sender = new User(-1); // placeholder user
   }
 
+  @computed
+  get parsedContent(): string {
+    return osu.linkify(this.content, true);
```

Comment:
should escape the content before passing to linkify (see comment on `discussionLinkify` function).